app/jobs/vpago/payment_capturer_job.rb

```diff
@@ -0,0 +1,8 @@
+module Vpago
+  class PaymentCapturerJob < ::ApplicationUniqueJob
+    def perform(payment_id)
```

Comment:
Let's give this queue a name, for instance, **payment_processing** and give it higher priority in the Sidekiq config 

```
  - [payment_processing, 3]
  - default
  ...
```